parcels/particlefile.py

```diff
@@ -3,12 +3,20 @@
 import netCDF4
 from datetime import timedelta as delta
 from parcels.tools.loggers import logger
+import os
+from tempfile import gettempdir
+import psutil
```

Comment:
psutil not found, both on my laptop and cartesius.
I don't know why this wasn't catched by travis.
Maybe another dependency needs it, I haven't updated my env recently. I'll try tomorrow

Reply:
Might be difference between anaconda and miniconda? In any case, I now added `psutil` to the dependency list